assets/lambda/new_dap_results_queue_notifier_lambda.py

```diff
@@ -12,28 +12,17 @@
 import json
 import logging
 import os
-import urllib.parse
 
 import boto3
 from botocore.exceptions import ClientError
+from capepy.aws.lambda_ import BucketNotificationRecord
+from capepy.aws.utils import decode_error
 
 logger = logging.getLogger(__name__)
 
 sqs_client = boto3.client("sqs")
```

Comment:
In a perfect world. The current capepy isn't as fully featured as it should be. This can either be something we do some planning and architecting before this is merged or we can do it after